catalog/core/catalog-core-downloadaction/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- /**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/ -->
+<blueprint xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.2.0"
+           xmlns:jaxrs="http://cxf.apache.org/blueprint/jaxrs"
+           xsi:schemaLocation="
+	    http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
+	    http://cxf.apache.org/blueprint/jaxrs http://cxf.apache.org/schemas/blueprint/jaxrs.xsd">
+
+    <bean id="resourceDownloadActionProvider"
+          class="org.codice.ddf.catalog.download.action.ResourceDownloadActionProvider">
+        <argument value="catalog.data.metacard.resource.download"/>
+    </bean>
+    
+    <service ref="resourceDownloadActionProvider" interface="ddf.action.ActionProvider">
+        <service-properties>
+            <entry key="id" value="catalog.data.metacard.resource.download"/>
+        </service-properties>
+    </service>
+
+    <bean id="resourceDownloadActionEndpoint" class="org.codice.ddf.catalog.download.action.ResourceDownloadActionEndpoint"/>
+
+	<jaxrs:server id="resourceDownloadEndpoint" address="/internal/catalog/download/cache">
```

Comment:
Do we need to restrict access to this new endpoint? Who should be allowed to call this?